01-exploring-pegasus.ipynb

```diff
@@ -1568,40 +1544,43 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# To see helper functions, select Jupyter File Explorer View from the Online Learning page\n",
-    "from helpers.problems import compare_embeddings\n",
-    "import numpy as np\n",
+    "average_chain = round(np.average([len(chain) for chain in sampleset.info['embedding_context']['embedding'].values()]), 1) \n",
+    "longest_chain = round(max([len(chain) for chain in sampleset.info['embedding_context']['embedding'].values()]), 1) \n",
+    "chain_breaks = round(100*np.average(sampleset.record.chain_break_fraction), 1) \n",
     "\n",
-    "compare_embeddings(samplesets)"
+    "print(\"Average chain length: {}.\".format(average_chain))\n",
+    "print(\"Longest chain: {}.\".format(longest_chain))\n",
```

Comment:
```suggestion
    "print(\"Average chain length: {:.1f}.\".format(average_chain))\n",
    "print(\"Longest chain: {}.\".format(longest_chain))\n",
```

or with an f-string: `print(f"Average chain length: {average_chain:.1f}.")`.

Reply:
You can apply the same logic to `chain_breaks` variable (i.e. round decimals for display only, not during calc).

Reply:
[Done too](https://github.com/dwave-examples/pegasus-notebook/pull/13/commits/43e05f8094fc48d1588933a6359df94861146a79), thanks: